snapper-samples/build.gradle

```diff
@@ -22,5 +22,5 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:23.0.1'
```

Comment:
How does it work if the project importing snapper uses a slightly different version of appcompat?